charts/sn-platform/templates/vault/vault-statefulset.yaml

```diff
@@ -32,9 +32,20 @@ spec:
   serviceAccount: {{ template "pulsar.vault.serviceAccount" . }}
   serviceType: {{ .Values.vault.serviceType }}
   statsdDisabled: true
+  {{- if .Values.vault.securityContext }}
+  securityContext: {{- toYaml .Values.vault.securityContext | nindent 4 }}
+  vaultEnvsConfig:
```

Comment:
Is this necessary?

Reply:
Without this config, seems the vault will be failed to start and print some `Operation not permitted` logs. 

I added these two Env configs from: https://github.com/hashicorp/docker-vault/issues/137#issuecomment-530247729

Reply:
specifying `securityContext` doesn't need to mean run as non-root user, so shall we just add a separate knob for these envs?